doc/build/faq/sessions.rst

```diff
@@ -395,7 +395,7 @@ setting a foreign-key attribute to a new value currently does not trigger
 an "expire" event of the :func:`_orm.relationship` in which it's involved.  This means
 that for the following sequence::
 
-    o = Session.query(SomeClass).first()
+    o = session.scalars(select(SomeClass)).first()
```

Comment:
**mike bayer** (zzzeek) wrote:

.limit(1)

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3729

Reply:
Changed